src/12-nextgen-projects/tasks.apib

```diff
@@ -12,6 +12,7 @@ Lists all the tasks that match the optional filters provided.
     + Attributes (object)
         + filter (object)
             + ids: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string])
+        + `page` (Page, optional)
```

Comment:
PP: By default, a parameter is optional, so the `optional` keyword could be dropped.